### PR TITLE
Eodp3 32: Detailer Dashboard Chart

### PR DESCRIPTION
**Pull request by P0und-cake**

### CODE REVIEW

File: apex-scripts/schedule-jobs.apex [hunk 1]
```diff
@@ -0,0 +1,6 @@
+BilletBatchClass billetJob = new BilletBatchClass();
```
Comment (P0und-cake) [inline apex-scripts/schedule-jobs.apex]: @tamilau You can open these scripts in illuminated cloud and run them if you ever need this. I tried to use the UI but it couldn't find your classes.

Comment (tamilau) [reply apex-scripts/schedule-jobs.apex]: Yeah, I didn't see them either until I changed them from global to public. Not sure why you aren't seeing them though.

File: src/aura/detailerDashboardMetrics/detailerDashboardMetrics.cmp [hunk 2]
```diff
@@ -2,105 +2,151 @@
...
+                        <div class="filter-error">{!'The number of months needs to be between ' + v.minMonthsToShow + ' and ' + v.maxMonthsToShow + '.'}</div>
+                    </aura:if>
+                </div>
+            </div>
+            <div class="slds-col slds-size--1-of-1">
+                <!-- We use the body tag to dynamically insert the detailerDashboardChart.component -->
+                <!-- This is necessary to redraw the chart when a user changes a filter. -->
+                <!-- To redraw the chart, we must first remove it from the page then add a new component. -->
+                <!-- If we don't do this, multiple charts get drawn in the same <canvas> tag causing the user to see many charts on top of each other -->
+                <!-- Which is bad. -->
```
Comment (tamilau) [inline src/aura/detailerDashboardMetrics/detailerDashboardMetrics.cmp]: This seems...obvious. (The last line, that is.)

Comment (P0und-cake) [reply src/aura/detailerDashboardMetrics/detailerDashboardMetrics.cmp]: I know, I was doing it for the lolz.

File: apex-scripts/schedule-jobs.apex (hunk 1 above)
Comment (P0und-cake) [reply apex-scripts/schedule-jobs.apex]: That is weird....you reduce the visibility and they are supposed to be available 🤔 

Oh well, no worries.

**Pull request merged**